### PR TITLE
rewrite from https://github.com/chris-steele/adapt-contrib-confidenceSlider

### PR DESCRIPTION
**Pull request by chris-steele**

### CODE REVIEW

**Comment by chris-steele** [comment]
To replace adapt-contrib-confidenceSlider and adapt-contrib-linkedConfidenceSlider

File: bower.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
 {
   "name": "adapt-contrib-confidenceSlider",
-  "version": "2.0.0",
+  "version": "2.1.1",
   "framework": "^2.0.0",
-  "homepage": "https://github.com/adaptlearning/adapt-contrib-confidenceSlider",
-  "issues": "https://github.com/adaptlearning/adapt-contrib-confidenceSlider/issues",
-  "repository": "git@github.com:adaptlearning/adapt-contrib-confidenceSlider.git",
-  "description": "A contributed two-part component that measures user confidence, using a sliding scale.",
+  "homepage": "https://github.com/chris-steele/adapt-contrib-confidenceSlider",
```
Comment (tomgreenfield) [inline bower.json]: Might want to purge your name from `homepage` and `issues` here. You know it'll just come back to bite you.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (tomgreenfield) [inline package.json]: Please ditch ```package.json```.

File: less/confidenceSlider.less [hunk 3]
```diff
@@ -1,151 +1,23 @@
...
+    			filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0);
+            	opacity: 0;
+    		}
+    		&.submitted {
+    			.rangeslider__fill {
+					filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=100);
+		        	opacity: 1;
+				}
+    		}
+    	}*/
```
Comment (tomgreenfield) [inline less/confidenceSlider.less]: Should this commented section be removed?

File: templates/confidenceSlider.hbs [hunk 4]
```diff
@@ -1,35 +1,31 @@
...
-                </div>
+    <div class="slider-widget component-widget {{#unless _isEnabled}} disabled {{#if _isComplete}} complete{{/if}}{{/unless}}">
+        <div class="slider-holder clearfix">
+            <div class="slider-scale-labels">
+                <div class="slider-scale-start" tabindex="0" role="region">{{labelStart}}</div>
+                <div class="slider-scale-end" tabindex="0" role="region">{{labelEnd}}</div>
+            </div>
+            <div class="slider-scale-numbers clearfix">
+                {{#each _items}}
+                    <button tabindex="0" role="button" class="base slider-scale-number" data-id="{{value}}">{{value}}</button>
```
Comment (tomgreenfield) [inline templates/confidenceSlider.hbs]: ```role="button"``` isn't needed here.

**Pull request merged**